src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/ProcessSamlAuthenticationHandler.php

```diff
@@ -159,6 +159,7 @@ public function process(GetResponseEvent $event)
                 return;
             }
 
+            $this->authenticatedSession->logAuthenticationMoment();
```

Comment:
good catch, checked it and indeed had a commit still local that added this (rebased a bit too much before pushing :flushed: ) - hidden between adding lots of debug log messages haha.